content/barbuddies/dante.md

```diff
@@ -2,13 +2,15 @@
 name: Dante
 pronouns: 
   nl: die/diens
-  en: they/them
 sites:
   - dwh
   - outsite
 nl:
-  Heeyhey, ik ben Dante. Ik ben 22 jaar oud en studeer Industrieel Ontwerp. Naast mijn studie vind ik het erg leuk om
-  saxofoon te spelen en loop ik vaak rond op DWH. Ik heb hier zelf helemaal mijn plek gevonden en wil ook graag
-  dat andere mensen zich hier thuis kunnen voelen dus heb ik mezelf opgegeven als barbuddy. Kom gezellig een praatje
-  maken en als je het leuk vind stel ik je ook voor aan een aantal andere DWHers.
+  Heeyhey, ik ben Dante. Ik ben 24, studeer Publieke Organisatie Management en woon in de top stad Delft. Naast mijn 
+  studie speel ik saxofoon en ben ik vaak te vinden op DWH.  Ik heb hier zelf helemaal mijn plek gevonden en wil ook 
+  graag dat andere mensen zich hier thuis kunnen voelen dus heb ik mezelf opgegeven als barbuddy. Kom gezellig een 
+  praatje maken en als je het leuk vind stel ik je ook voor aan een aantal andere DWH'ers.
+
```

Comment:
ik denk niet dat dit rendert als new lines

Reply:
Removed the empty line